### DISCUSSION TITLE
yalmip error code -10

### DISCUSSION THREAD

**Discussion by PureStudent** [yalmip/YALMIP#1411] [Problems using YALMIP] [answered]
Hello prof.
I used the command ’optimizer‘ to encapsulate the model, and then called the solution. The values are NAN, and the error code is -10.
In this website:"https://yalmip.github.io/command/yalmiperror/" I only see yalmiperror  Error codes from -9 to 17. So what's the meaning    of error code -10?

**Comment by johanlofberg** [comment, marked as answer]
you've sent nan to the optimizer

`
>> yalmiperror
    -12 Solver license cannot be located 
    -11 Solver license expired 
    -10 NaN in model data (learn to debug)
    -9 Specified solver not recognized 
    -8 Problem does not satisfy geometric programming rules
    -7 Solver does not return error codes 
    -6 Search space not bounded 
    -5 License problems in solver 
    -4 Solver not applicable 
    -3 Solver not found 
    -2 No suitable solver
    -1 Unknown error (learn to debug) 
    0 Successfully solved 
    1 Infeasible problem (learn to debug) 
    2 Unbounded objective function (learn to debug) 
    3 Maximum iterations or time limit exceeded 
    4 Numerical problems (learn to debug) 
    5 Lack of progress (learn to debug)
    6 Initial solution infeasible 
    7 YALMIP called solver with incorrect input 
    8 Feasibility cannot be determined 
    9 Unknown problem (learn to debug) 
    10 bigM failed, increase sp.Mfactor 
    11 Other identified error 
    12 Either infeasible or unbounded (learn to debug) 
    13 YALMIP cannot determine status in solver 
    14 Model creation failed (learn to debug)
    15 Infeasible or unbounded problem (learn to debug) 
    16 User terminated 
    17 Presolve recovery failed 
    18 Missing non-negativity bounds in GP formulation 
    19 Convexity requirements not met 
    20 Solver complains about bad data (learn to debug) 
    21 Failed to initialize bisection space (probably infeasible)
    22 Ill-posed problem according to solver 
    23 Unsupported mixed cone 
    24 Infeasible in presolve 
`

**Reply by PureStudent** [reply]
Thanks！
